06-building-a-program-start-to-finish/solving-the-problem/output/invertedindexutil.js

```diff
@@ -0,0 +1,276 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I believe you can have all your major functions in InvertedIndex.js. Just seems like the invertedIndex class is doing little. Please address.